ToyBox/classes/MonkeyPatchin/BagOfPatches/LevelUpPatchesWrath.cs

```diff
@@ -487,12 +487,31 @@ private static class CharGenClassSelectorItemVM_SpecialRequiredRace_Patch {
             }
         }
 
+        [HarmonyPatch(typeof(BlueprintCharacterClass))]
+        private static class BlueprintCharacterClass_Patch {
+            [HarmonyPatch(nameof(BlueprintCharacterClass.MeetsPrerequisites)), HarmonyPostfix]
+            public static void MeetsPrerequisites_Patch(ref bool __result, BlueprintCharacterClass __instance) {
+                if (settings.toggleIgnoreFeaturePrerequisitesWhenChoosingClass &&
+                    !__instance.PrestigeClass && !__instance.IsMythic) {
+                    __result = true;
+                }
+            }
+        }
 
-        [HarmonyPatch(typeof(PrerequisiteFeature))]
+        [HarmonyPatch(typeof(BlueprintArchetype))]
+        private static class BlueprintArchetype_Patch {
+            [HarmonyPatch(nameof(BlueprintArchetype.MeetsPrerequisites)), HarmonyPostfix]
+            public static void MeetsPrerequisites_Patch(ref bool __result, BlueprintArchetype __instance) {
+                if (settings.toggleIgnoreFeaturePrerequisitesWhenChoosingClass )
+                    __result = true;
+                }
+            }
+
+        /*[HarmonyPatch(typeof(PrerequisiteFeature))]
```

Comment:
Since you're removing this patch; did you test that it still works?

Reply:
Yup, works correctly (actually looks like it works better, since Cleric's one archetype wasn't able to be selected with this, and is now able to be selected)